OpenRA.Game/Traits/Player/FrozenActorLayer.cs

```diff
@@ -38,7 +38,7 @@ public class FrozenActor
 		public int HP;
 		public DamageState DamageState;
 
-		public bool Visible;
+		public bool Visible = true;
```

Comment:
You can't just flip this. This changes behavior engine wide. The most noticeable regressions is that `FrozenUnderFog.StartsRevealed` (e.g. tech buildings) is now ignored.